grails-app/controllers/au/org/ala/layers/ShapesController.groovy

```diff
@@ -359,32 +367,29 @@ class ShapesController {
         render processWKTRequest(request.JSON, pid, namesearch) as JSON
     }
 
+    /**
+     * TODO
```

Comment:
Did you mean to add JavaDoc and forget? Either leave blank or fill it out.

Reply:
Forgot to remove 'todo' after it is done